ci-operator/config/devfile/integration-tests/devfile-integration-tests-main__v4.15.console.yaml

```diff
@@ -40,7 +40,7 @@ resources:
       memory: 8Gi
 tests:
 - as: e2e-aws-console-periodic
-  cron: 12 0 3,19 * *
+  cron: 0 2 * * 1-5
```

Comment:
Use the old cron configuration for this job. It'll reduce how often the (currently) broken tests run

Reply:
fixed in https://github.com/openshift/release/pull/53140/commits/54db7b92113f19e23f7d85c21f88e565c5bc6752